.github/workflows/main.yml

```diff
@@ -35,8 +39,10 @@ jobs:
       - name: Run ${{ matrix.suite-name }}
         env:
           TEST_PATTERN: ${{ matrix.suite-name }}
+          TEST_TARGET: ${{ matrix.sbt-target }}
         run: |
-          echo $TEST_PATTERN
+          echo "TEST_PATTERN=$TEST_PATTERN"
```

Comment:
The output of this line just repeats the name of the test stage. I think it's redundant.